source/Calamari.Tests/Fixtures/ConfigurationTransforms/TransformFileLocatorFixture.cs

```diff
@@ -352,7 +352,7 @@ public void When_TransformIsWildcardFileNameOnly_And_TargetIsFileNameOnly_2_ItSu
         }
 
         [Test]
-        public void When_TransformIsWildcardFileNameOnly_And_TargetIsWildCardInMiddleOfFileNameOnly_ItFails()
+        public void When_TransformIsWildcardFileNameOnly_And_TargetIsWildCardInsideFileNameOnly_ItFails()
```

Comment:
I renamed these as they were causing issues with too long path names for me locally during clean of tests, my path is `C:\src\OctopusDeploy\Calamari\...` so I don't think its unreasonably long, not sure if others have come across this.